language/compiler/src/unit_tests/testutils.rs

```diff
@@ -34,7 +34,7 @@ macro_rules! instr_count {
 fn compile_script_string_impl(
     code: &str,
     deps: Vec<CompiledModule>,
-) -> Result<(CompiledScript, Vec<VMStatus>)> {
+) -> Result<(CompiledScript, Option<VMStatus>)> {
```

Comment:
just as a random comment... there is no reason for this and related not to be a `VMResult` or `Result<CompiledScript, VMStatus>`, or is it?

Reply:
The test call sites always need the compiled script back. The outer result is for file reading stuff